modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -1534,7 +1534,7 @@ void LMDBBackend::lookup(const QType& type, const DNSName& qdomain, int zoneId,
     d_matchkey = co(zoneId, relqname, type.getCode());
```

Comment:
this case (including type) does not even need to be a prefix query - a simple `get` by this key would yield the same result. But, after this PR, I'm not sure it matters much.